code/modules/mechs/equipment/utility.dm

```diff
@@ -48,8 +54,13 @@
 				owner.visible_message(SPAN_NOTICE("\The [owner] begins loading \the [O]."))
 				playsound(src, 'sound/mechs/hydraulic.ogg', 50, 1)
 				if(do_after(owner, 20, O, 0, 1))
+					if(O in carrying || O.buckled_mob || O.anchored || (locate(/mob/living) in O)) //Repeat checks
```

Comment:
damn , not letting people pick up closets full of people so they can unleash a minion army of excel against IH ? lame.

Reply:
Shrug. It's standard Bay stuff, don't know if it was possible before.